README.md

```diff
@@ -78,9 +78,12 @@ The `revenue` command creates an earnings report for the interval you specify:
 - `urcli revenue` - Returns two reports; one for your total earnings and one for the earnings of the current month you are in.
 - `urcli revenue 2015-11-24` - Generates an earnings report for that date.
 - `urcli revenue 2016-05` - Generates an earnings report for that month.
+- `urcli revenue 2016` - Generates an earnings report for that year.
 - `urcli revenue 1` - (You can use the month numbers, 1-12, as periods). This command will generate a report for the whole month of January of the current year. Tip: If you input a future month it will generate a report for the same month of the previous year.
 - `urcli revenue today` - Generates an earnings report for today.
 - `urcli revenue yesterday` - Generates an earnings report for yesterday.
+- `urcli revenue month` - Generates an earnings report for current month.
+- `urcli revenue year` - Generates an earnings report for current year.
```

Comment:
I actually disagree with you on this one. My reason is that they aren't shortcuts. If you want the current month, it's easier to write `urcli revenue 3` than `urcli revenue month`. With year it's more or less the same.

Also, I think there is a chance of user confusion here. When you see that the arguments are just called `month`  and `year`, users may read it as if they have to use the before the year/month argument, like so `urcli revenue month 3`. And it would be crazy to create `current-month` or `this-month` as arguments.

The only reason I included `today` and `yesterday` is that they are much easier to write than a date. You may even have to remember what date it is and so on, but with `today` you don't have to think about anything. This is not the case with `month` and `year`. No one has to look up what year it is :smile:.